README.md

```diff
@@ -52,6 +52,21 @@ You can access the following commands from the [Visual Studio Code command palle
 | ROS: Preview URDF | Preview URDF and Xacro files. Updates after changes are saved. |
 | ROS: Install ROS Dependencies for this workspace using rosdep | Shortcut for `rosdep install --from-paths src --ignore-src -r -y`. |
 
+## Launch Debugging
+The Visual Studio Code extension for ROS supports launch debugging for ROS 1 and ROS 2 nodes. The VSCode extension currently supports debugging ROS written in Python and C++, and the nodes must be placed in a ROS launch file with the extension `.launch` or `.py`.
```

Comment:
[nit] Could split up the sentence:
The VSCode extension currently supports debugging ROS **nodes** written in Python and C++. The nodes must be placed in a ROS launch file with the extension `.launch` or `.py`.